catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -507,4 +508,25 @@ public static int getBit(int number, int index) {
 		}
 		return 0;
 	}
+
+	public static String stringWithoutTrailingZero(String value) {
+		String returnedValue;
+		if (value.contains(".")) {
+			try {
+				String[] splitter = value.split("\\.");
+				int afterDecimalCount = splitter[1].length();
+				String pattern = "#.";
+				for (int i = 0; i < afterDecimalCount; i++) {
```

Comment:
is there going to be any limit on how many decimal places there should be displayed?